api/server/admin/service.go

```diff
@@ -1,13 +1,15 @@
 package admin
 
 import (
+	adminPb "github.com/textileio/powergate/api/gen/powergate/admin/v1"
 	"github.com/textileio/powergate/ffs/manager"
 	"github.com/textileio/powergate/ffs/scheduler"
 	"github.com/textileio/powergate/wallet"
 )
 
 // Service implements the Admin API.
 type Service struct {
+	adminPb.UnimplementedAdminServiceServer
```

Comment:
Required by the new version of gRPC to help with evolving the API moving forward. That type just provides default implementations of the service that returning `unimplemented` errors.